spec/helpers/application_helper_spec.rb

```diff
@@ -48,4 +48,13 @@
       end
     end
   end
+
+  describe "#seo_canonical_url" do
```

Comment:
Rather than I helper spec, it's probably more valuable writing a feature spec to assert the tag is rendered in the markup.

Reply:
cool thanks, since we're having the google indexing issue I'll merge this now and revisit the spec in a separate PR.